rules/opentitan/sim_verilator.bzl

```diff
@@ -113,6 +138,12 @@ def _test_dispatch(ctx, exec_env, provider):
 
     # Get the pre-test_cmd args.
     args = get_fallback(ctx, "attr.args", exec_env)
+    if ctx.attr.kind == "rom":
+        # FIXME: This is a bit of inside-baseball: We know the opentitantool
```

Comment:
This might be more complicated but maybe the param dict should have two test commands, one for flash-based tests and one for rom-based tests? Say, if `args` is an array, it is valid for all kind, but it can also be a dictionary of key/values where the keys are the `kind`.

Reply:
I have two alternate ideas to explore rather than making the test_env more complex:
- Allow opentitantool to accept an empty parameter for `--verilator-flash`, meaning there is no flash payload.  This is the current behavior with `--verilator-{rom,otp}` and used to be the behavior with `flash`.  It changed when opentitantool was updated to accept multiple flash args to load multiple banks.
- Provide an "empty" VMEM file to load into flash.